packaging_assets/package.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+# Copyright 2022 Contributors to the Parsec project.
+# SPDX-License-Identifier: Apache-2.0
+
+# Create a quickstart package
+
+ARCH="x86"
+OS="linux"
+PACKAGE_PATH=$(pwd)
+WORK_DIR="/tmp/workdir-parsec/"
+JOBS_NUMBER=1
+SCRIPT_PATH=$(realpath $0)
+ASSETS_DIR=$(dirname $SCRIPT_PATH)
+PARSEC_DIR=$(dirname $ASSETS_DIR)
+VERSION=$(cargo metadata --format-version=1 --no-deps --offline | python3 -c "import sys, json; data=json.load(sys.stdin); print([pkg['version'] for pkg in data['packages'] if pkg['name'] == 'parsec-service'][0])")
+
+# Usage
+USAGE_STR=\
+"Usage:\n"\
+"package.sh [Options]\n"\
+"Options:\n"\
+" -o {path}: Output absolute path, the default path is the current directory i.e. $(pwd)\n"\
+" -j {jobs}: Number of parallel jobs, Default is $JOBS_NUMBER"\
+" -h : Display this help menu\n"
+
+# Flags
+while getopts v:o:j:h flag
+do
+    case "${flag}" in
+        o) PACKAGE_PATH=${OPTARG};;
+        j) JOBS_NUMBER=${OPTARG};;
+        h) echo -e $USAGE_STR; exit 0;;
+    esac
+done
+
+check_mandatory_flags() {
+    echo "Checking mandatoy flags..."
+    if [ -z "$VERSION" ];then
+            echo 'Missing the version please run with -h for help' >&2
+            exit 1
+    fi
+}
+
+check_release_tag() {
+    CURRENT_TAG=$(git name-rev --tags HEAD | cut -d " " -f 2)
+    LATTEST_TAG=$(git tag | tail -n 1)
+    if [ -z "$LATTEST_TAG" ];then
+        echo "Warning:No tags"
+    fi
+    if [ "$LATTEST_TAG" == "$CURRENT_TAG" ]; then
+        echo "Packaging release tag: $LATTEST_TAG"
+    else
+        echo "Warning: The current HEAD isn't tagged"
+        echo "Warning: Please checkout the latest tag : $LATTEST_TAG"
+        read  -n 1 -p "Do you want to continue anyway [y/n]?" choice
+        if [ "$choice" != "y" ]; then
+            exit 1
+        fi
+    fi
+}
+
+cleanup()
+{
+    echo "Clean up"
+    rm -rf $WORK_DIR/$PACKAGE_DIR
+    rm -rf $WORK_DIR
+}
+
+pre_package() {
+    # Construct package name
+    PACKAGE_DIR=quickstart-$VERSION-${OS}_$ARCH
+
+    # Create a temp work directory
+    mkdir -p $WORK_DIR
```

Comment:
This is not required because `mkdir -p $WORK_DIR/parsec` below will create $WORK_DIR